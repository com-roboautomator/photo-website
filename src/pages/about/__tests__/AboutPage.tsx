import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import '@testing-library/jest-dom/extend-expect'
import AboutPage from '../AboutPage'

describe('About Page', () => {

    it('should match snapshot', () => {
        const ImageSliderRender = renderer
            .create(
                <AboutPage />
            )
            .toJSON()
        expect(ImageSliderRender).toMatchSnapshot()
    })

    it('should render correctly', () => {

        const { getByTestId } = render(<AboutPage />)

        expect(getByTestId('PageTitle-Container')).toBeInTheDocument()
        expect(getByTestId('About-Page')).toBeInTheDocument()
        expect(getByTestId('ProfileImage-Container')).toBeInTheDocument()
        expect(getByTestId('About-Page-Text')).toBeInTheDocument()

    })

})