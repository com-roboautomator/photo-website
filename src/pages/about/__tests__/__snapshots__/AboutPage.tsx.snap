// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page should match snapshot 1`] = `
<main>
  <div
    className="PageTitle-container-5"
    data-testid="PageTitle-Container"
  >
    <img
      alt=""
      className="PageTitle-image_loading-7"
      data-testid="PageTitle-Image"
      onLoad={[Function]}
      src="https://picsum.photos/id/10/1500/200"
      style={
        Object {
          "height": "20vh",
        }
      }
    />
    <div
      className="PageTitle-title_loading-8"
      data-testid="PageTitle-Text"
    >
      ABOUT
    </div>
    <div
      className="PageTitle-signature-10"
    >
      <img
        alt=""
        data-testid="PageTitle-Signature"
        src=""
        width="25%"
      />
    </div>
  </div>
  <div
    className="About-container-1"
    data-testid="About-Page"
  >
    <div
      className="About-wrapper-2"
    >
      <div
        className="About-profile_image-3"
      >
        <div
          className="ProfileImage-container-11"
          data-testid="ProfileImage-Container"
        >
          <img
            alt=""
            className="ProfileImage-image-12"
            data-testid="ProfileImage-Image"
            src="SaraDoneuxImage.jpg"
          />
          <div
            className="ProfileImage-text-13"
            data-testid="ProfileImage-Text"
          >
            Sara Doneux
          </div>
        </div>
      </div>
      <div
        className="About-text-4"
        data-testid="About-Page-Text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Eu scelerisque felis imperdiet proin. Pretium aenean pharetra magna ac. Scelerisque fermentum dui faucibus in ornare quam viverra orci. Id porta nibh venenatis cras sed felis eget. Pharetra pharetra massa massa ultricies. Metus dictum at tempor commodo ullamcorper. Leo urna molestie at elementum eu facilisis. Faucibus turpis in eu mi bibendum neque egestas congue. Augue ut lectus arcu bibendum at. Sed adipiscing diam donec adipiscing tristique risus. Dignissim cras tincidunt lobortis feugiat vivamus at augue eget. Ullamcorper eget nulla facilisi etiam dignissim. Rutrum quisque non tellus orci ac auctor. Massa tincidunt dui ut ornare lectus sit amet. Blandit aliquam etiam erat velit scelerisque in dictum non. Eleifend mi in nulla posuere sollicitudin.

Purus sit amet luctus venenatis lectus magna. A diam sollicitudin tempor id eu nisl nunc mi. Mattis molestie a iaculis at. Sed odio morbi quis commodo odio aenean sed adipiscing. Sed vulputate mi sit amet mauris commodo. Habitant morbi tristique senectus et netus. Donec adipiscing tristique risus nec feugiat in. Egestas sed sed risus pretium quam vulputate dignissim. Orci dapibus ultrices in iaculis. Scelerisque mauris pellentesque pulvinar pellentesque habitant morbi tristique senectus et. Elementum nibh tellus molestie nunc non.

Tellus rutrum tellus pellentesque eu tincidunt tortor. Pellentesque id nibh tortor id aliquet lectus proin nibh. Enim ut tellus elementum sagittis vitae et leo. In ornare quam viverra orci sagittis eu volutpat odio facilisis. Eget nunc scelerisque viverra mauris in aliquam sem fringilla ut. Feugiat in ante metus dictum at tempor. Nullam vehicula ipsum a arcu cursus vitae congue. Eget duis at tellus at urna condimentum mattis pellentesque id. Enim ut sem viverra aliquet eget sit. Faucibus interdum posuere lorem ipsum dolor sit amet. Sit amet nisl purus in. Egestas congue quisque egestas diam in arcu.
      </div>
    </div>
  </div>
</main>
`;
